From 6614bb62ecbec7c3b18ea7fe44beb50fe7942b27 Mon Sep 17 00:00:00 2001
From: Jan Beich <jbeich@FreeBSD.org>
Date: Thu, 20 Jan 2022 01:27:48 +0000
Subject: [PATCH] meson: drop unused argument for i18n.merge_file()

Ignored in Meson < 0.60.0, deprecated since 0.60.1 and fatal since 0.61.0.

podcasts-gtk/resources/meson.build:5:0: ERROR: Function does not take positional arguments.
podcasts-gtk/resources/meson.build:30:0: ERROR: Function does not take positional arguments.
---
 podcasts-gtk/resources/meson.build | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/podcasts-gtk/resources/meson.build b/podcasts-gtk/resources/meson.build
index c1f1ee9f..87148087 100644
--- a/podcasts-gtk/resources/meson.build
+++ b/podcasts-gtk/resources/meson.build
@@ -2,7 +2,7 @@ subdir('icons')
 
 desktop_conf = configuration_data()
 desktop_conf.set('icon', application_id)
-desktop_file = i18n.merge_file ('desktop-file',
+desktop_file = i18n.merge_file (
   type: 'desktop',
   input: configure_file(
     input: files('org.gnome.Podcasts.desktop.in.in'),
@@ -27,7 +27,7 @@ endif
 
 appdata_conf = configuration_data()
 appdata_conf.set('appid', application_id)
-appdata_file = i18n.merge_file ('appdata-file',
+appdata_file = i18n.merge_file (
   input: configure_file(
     input: files('org.gnome.Podcasts.appdata.xml.in.in'),
     output: 'org.gnome.Podcasts.appdata.xml.in',
-- 
GitLab

